Tests/GRPCNIOTransportCoreTests/Client/GRPCClientStreamHandlerTests.swift

```diff
@@ -40,10 +40,9 @@ final class GRPCClientStreamHandlerTests: XCTestCase {
     let framesToBeIgnored: [HTTP2Frame.FramePayload] = [
       .ping(.init(), ack: false),
       .goAway(lastStreamID: .rootStream, errorCode: .cancel, opaqueData: nil),
-      // TODO: uncomment when it's possible to build a `StreamPriorityData`.
-      // .priority(
-      //   HTTP2Frame.StreamPriorityData(exclusive: false, dependency: .rootStream, weight: 4)
-      // ),
+      .priority(
+        HTTP2Frame.StreamPriorityData(exclusive: false, dependency: .rootStream, weight: 4)
```

Comment:
Do we need to bump the minimum required http2 version as well?

Reply:
No, we're already on the latest version